sigal/gallery.py

```diff
@@ -371,7 +379,7 @@ def thumbnail(self):
                     except:
                         self.logger.error("Failed to open %s", f.src_path)
                     else:
-                        if im.size[0] > im.size[1]:
+                        if im.size[0] > im.size[1]: 
```

Comment:
I have tried but I am a python "hacker" (bidouilleur) and did not succeed (got some errors with tests). Sorry about that.